packages/form-js-editor/src/render/Renderer.js

```diff
@@ -0,0 +1,61 @@
+import { render } from 'preact';
+import { useState } from 'preact/hooks';
+
+import FormEditor from './components/FormEditor';
+
+import { FormEditorContext } from './context';
+
+/**
+ * @typedef { { container } } Config
+ * @typedef {any} EventBus
```

Comment:
Proposal: Import eventBus that is contained / exported from this library:


```suggestion
 * @typedef { import('../core/EventBus').default } EventBus
```

Reply:
cf. https://github.com/bpmn-io/form-js/pull/75#discussion_r644824394